packages/server/src/utils/index.ts

```diff
@@ -952,3 +954,29 @@ export const getAllValuesFromJson = (obj: any): any[] => {
     extractValues(obj)
     return values
 }
+
+export const replaceEnvVariables = async (question: string, appDataSource: DataSource): Promise<string> => {
```

Comment:
I think we can delete this right? It was a misunderstanding to have the env variables enabled in question

Reply:
actually hold on, we can reuse this